tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -290,9 +290,81 @@ func (a *AwsProvider) findOrCreateSg(infraID string, vpc *ec2.Vpc) (string, erro
 	if err != nil {
 		return a.createWindowsWorkerSg(infraID, vpc)
 	}
+
+	log.V(0).Info(fmt.Sprintf("Using existing Security Group: %s.", sgID))
```

Comment:
I followed the log method as the other log in the file. I don't know about the log decision.

Reply:
I didn't know about any decision to change back from `log.V(0).Info()` to `log.Info()`. Essentially they are the same at this stage according to the documentation. We decided to go with `log.V(0).Info()` because we have implementations to set logs to other levels if necessary (https://github.com/openshift/windows-machine-config-operator/blob/e3e85583d88907291692faf39521b28acfc688b8/log/log.go#L11) for example "debug", "info", "warn". etc, and I am not aware of any further discussion. I think we should maintain the level.

Reply:
@Bowenislandsong we need to re-discuss this again.

Reply:
Is this one resolved?

Reply:
@leonjia0112 please use `log.Info()`